### ISSUE TITLE
Formatting messed up after update to 2.2.5

### ISSUE THREAD

**Issue by jneubert** [Triply-Dev/YASGUI.YASQE-deprecated#44]
After updating from YASQE 2.2.4 [example page](http://zbw.eu/beta/sparql-lab/index.html?queryRef=https://api.github.com/repos/jneubert/skos-history/contents/sparql/added_concepts.rq) to 2.2.5 [same example page](http://zbw.eu/beta/sparql-lab/test1.html?queryRef=https://api.github.com/repos/jneubert/skos-history/contents/sparql/added_concepts.rq), formatting changes (different spacing of lines, settings such as  "viewportMargin: Infinity" are ignored).

Supposedly related to https://github.com/YASGUI/YASR/issues/23

**Comment by LaurensRietveld** [comment]
I believe I fixed this a few days ago (but forgot to push it as new version). Could you check the newest version I published?

Reason for bug: scoping of all css caused the top-level css attributes (i.e. directly assigned to yasqe/codemirror class) to be ignored

**Comment by jneubert** [comment]
With 2.2.6 I've the same effect.The "height:auto" property of the .CodeMirror class assigned in a style element in index.html (and supposedly the other properties defined there) seems to get lost. This looks like a very good fit with your description above.

**Comment by LaurensRietveld** [comment]
Ah, yes, I see what's going on. Because I've scoped all the styles with the .yasr class, the regular CodeMirror styles are now defined with css rules such as '.yasqe .CodeMirror {border: .....}'. (instead of the same, without the .yasqe class selector)

In the older version, your custom styles got precedence. In the new verison, the newer codemirror styles are more specific than yours (.yasqe .CodeMirror vs. .CodeMirror), so you do not get precedence.
The fix is easy enough, by making your css selector more specific (e.g. '.yasqe div.CodeMirror { border ...}')

So, in short: this behaviour is intended.
But, in hind sight, the version tag for this change should have been major instead of minor, as other users might have experienced similar issues.

**Comment by jneubert** [comment]
That was an easy fix. Thanks for the explanation.
